compose/seatable-server.yml

```diff
@@ -34,6 +34,8 @@ services:
       - SEATABLE_ENABLE_CREATE_BASE_FROM_TEMPLATE=${SEATABLE_ENABLE_CREATE_BASE_FROM_TEMPLATE:-}
       - SEATABLE_HELP_LINK=${SEATABLE_HELP_LINK:-https://docs.seatable.io}
       - SEATABLE_LOG_LEVEL=${SEATABLE_LOG_LEVEL:-INFO}
+      - ENABLE_SEADOC=${ENABLE_SEADOC:-false}
+      - SEADOC_SERVER_URL=${SEADOC_SERVER_URL:-https://seadoc.example.com:7070}
```

Comment:
I would prefer `:?Variable is not set or empty` instead of the default value (which will never work)